src/fonts/Makefile

```diff
@@ -0,0 +1,139 @@
+#!gmake
```

Comment:
Two Makefiles https://github.com/KaTeX/katex-fonts/blob/master/src/Makefile and https://github.com/KaTeX/katex-fonts/blob/master/src/fonts/Makefile were merged to one and paths were updated accordingly